sdk/storage/azure-storage-blobs/test/block_blob_client_test.cpp

```diff
@@ -97,7 +97,8 @@ namespace Azure { namespace Storage { namespace Test {
         std::vector<uint8_t>(
             m_blobContent.begin() + static_cast<std::size_t>(options.Range.GetValue().Offset),
             m_blobContent.begin()
-                + static_cast<std::size_t>(options.Range.GetValue().Offset + options.Range.GetValue().Length.GetValue())));
+                + static_cast<std::size_t>(
+                    options.Range.GetValue().Offset + options.Range.GetValue().Length.GetValue())));
```

Comment:
Should we come into agreement on the version of clang-format we use so as to avoid this kind of changes in the future?

Reply:
I thought we did. Anything 9 or above. I am using `10.0.0`. What version were you using?
https://github.com/Azure/azure-sdk-for-cpp/blob/910731893a2c113392f3be1979045cc9c5847bae/CONTRIBUTING.md#third-party-dependencies